x-pack/legacy/plugins/snapshot_restore/server/routes/api/repositories.ts

```diff
@@ -34,14 +39,19 @@ export function registerRepositoriesRoutes(router: Router, plugins: Plugins) {
   router.delete('repositories/{names}', deleteHandler);
 }
 
+interface ManagedRepository {
+  name?: string;
+  policy?: string;
+}
+
 export const getAllHandler: RouterRouteHandler = async (
-  req,
+  _req,
```

Comment:
Just curious, why was this renamed?

Reply:
Ah, I think this might have been auto-fixed by my editor since it is unused. I reverted the change.